src/to-have-style.js

```diff
@@ -48,8 +48,19 @@ function expectedDiff(expected, computedStyles) {
   return diffOutput.replace(`${chalk.red('+ Received')}\n`, '')
 }
 
-export function toHaveStyle(htmlElement, css) {
+function getCss(document, css) {
+  if (typeof css === 'object') {
+    const sandboxElement = document.createElement('div')
+    Object.assign(sandboxElement.style, css)
+    return sandboxElement.style.cssText
+  }
```

Comment:
Wow, this was surprisingly simple. Looking forward to have this PR marked as ready.